contract/Lykke.Job.SiriusCashoutProcessor.Contract/Events/CashoutCompletedEvent.cs

```diff
@@ -8,6 +8,7 @@ public class CashoutCompletedEvent
     {
         public Guid OperationId { get; set; }
         public Guid ClientId { get; set; }
+        public Guid? WalletId { set; get; } // null if API wallet cashout
```

Comment:
Shouldn't it be vice versa? I mean the comment. The value is null if it's a trading wallet cashout